src/plugins/virtualbox/vm.cpp

```diff
@@ -0,0 +1,121 @@
+// albert - a simple application launcher for linux
```

Comment:
The name of the class is missleading. I first thought this is the item that represents the vm and the vmitem is a listadapter or sth like that. Basically this is a VBoxManage output parser. But I think it is not really necessary to have a dedicated  class for this, since this is plain imperative code only.